babel.config.js

```diff
@@ -25,6 +25,8 @@ const config = {
 		],
 		'@babel/react',
 	],
+	// Exclude transforms that make all code slower, see https://github.com/facebook/create-react-app/pull/5278
+	exclude: [ 'transform-typeof-symbol' ],
```

Comment:
This should be an option for `@babel/env` rather than a property of the Babel config itself. I'll push a fix in a second.